pkg/operator/certmonitor/monitor.go

```diff
@@ -0,0 +1,196 @@
+package certmonitor
```

Comment:
if it's going to be tightly coupled, move it as a subpackage of the main controller